packages/jsii-pacmak/lib/reflect-hacks.ts

```diff
@@ -0,0 +1,28 @@
+import * as reflect from 'jsii-reflect';
+import * as spec from 'jsii-spec';
+
+// All current functions are written against jsii-spec, but jsii-reflect
+// is more convenient.
+//
+// These functions exist to break the isolation barrier until we have time
+// to rewrite the code to properly use jsii-reflect.
+
```

Comment:
Why not just expose spec as public in Jsii-reflect?

Reply:
I went down that path first, but I think it's the wrong way round. Don't feel that spec should be exposed.